code/modules/organs/limbs.dm

```diff
@@ -937,7 +937,8 @@ Note that amputating the affected organ does in fact remove the infection from t
 			spark_system.set_up(5, 0, owner)
 			spark_system.attach(owner)
 			spark_system.start()
-			QDEL_NULL_IN(src, spark_system, 1 SECONDS)
+			spawn(1 SECONDS)
```

Comment:
`INVOKE_NEXT_TICK` instead, to a proc that does the QDEL_NULL.

Reply:
Actually just a callback.

Reply:
It can't be a callback though, that's the entire reason I'm making this PR